### PR TITLE
Allow for mutliple args on emit

### PR DESCRIPTION
**Pull request by toddself**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -139,14 +144,17 @@ Nanobus.prototype._emit = function (arr, eventName, data, uuid) {
     data = eventName
     eventName = null
   }
+
   var length = arr.length
+  var args = data
```
Comment (yoshuawuyts) [inline index.js]: don't think this is necessary

Comment (toddself) [inline index.js]: that line is how the default use case of `listener.apply(listener, data)` gets set up. Only in the case where `eventName` is set, does it alter the list of arguments.

Comment (toddself) [inline index.js]: i could alter args in place, but the first clause in the if statement there is pre-pending the eventName.  `shift` tends to be more expensive than just re-binding to a new variable.

**Comment by toddself** [comment]
@yoshuawuyts are ok if we publish this as a minor?

**Comment by yoshuawuyts** [comment]
`v4.3.0` :tada:

**Pull request merged**